ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+set -e
+
+. util/build_consts.sh
+SHA=$(git rev-parse HEAD)
+BITCACHE_DIR=~/.cache/opentitan-bitstreams/cache/${SHA}
+mkdir -p $BITCACHE_DIR
+BITCACHE_FILE=$BITCACHE_DIR/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.orig
+cp $BIN_DIR/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit ${BITCACHE_FILE}
+echo "" >> ${BITCACHE_DIR}/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.splice
+echo -n ${SHA} > ${BITCACHE_DIR}/../../latest.txt
+export BITSTREAM="--offline --list ${SHA}"
+
+# TODO: remove the --notest_keep_going and --nokeep_going flag after the CW310
+# tests are reliable (#13044)
+ci/bazelisk.sh test \
+    --define DISABLE_VERILATOR_BUILD=true \
```

Comment:
More Nits, just things you should know they don't need fixing here: This isn't necessary, this shouldn't include the verilator tests anymore